x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -15,32 +15,195 @@ const END_DATE = moment.utc(DATES.metricsAndLogs.hosts.max);
 const timepickerFormat = 'MMM D, YYYY @ HH:mm:ss.SSS';
 
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const kibanaServer = getService('kibanaServer');
   const esArchiver = getService('esArchiver');
   const browser = getService('browser');
-  const pageObjects = getPageObjects(['common', 'infraHome', 'timePicker', 'infraHostsView']);
-  const kibanaServer = getService('kibanaServer');
+  const security = getService('security');
+  const pageObjects = getPageObjects([
+    'common',
+    'infraHome',
+    'timePicker',
+    'infraHostsView',
+    'security',
+    'settings',
+  ]);
+
+  // Helpers
```

Comment:
I added some methods before the tests to avoid duplicating the logic in the tests